libs/core-ui/src/lib/components/InteractiveLegendUtils.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { interactiveLegendStyles } from "./InteractiveLegend.styles";
+
+export function getColorBoxClassName(
+  index: number,
+  color?: string,
+  activated?: boolean
+): string {
+  const classes = interactiveLegendStyles(activated, color);
+  // this is used as data series symbol in the side panel, the sequence needs to be consist with the sequence of symbols in ScatterUtils getScatterSymbols()
+  switch (true) {
+    case index % 5 === 1:

```

Comment:
nit: do the index % 5 outside of the switch case, so:
```
modIndex = index % 5
switch (modIndex) {
    case 1:
       ...
    case 2:
       ....
```

Reply:
updated thanks!